packages/plugins/geth/src/index.js

```diff
@@ -7,18 +7,17 @@ import DevTxs from "./devtxs";
 const constants = require('embark-core/constants');
 
 class Geth {
-  constructor(embark, options) {
+  constructor(embark) {
     this.embark = embark;
     this.embarkConfig = embark.config.embarkConfig;
     this.blockchainConfig = embark.config.blockchainConfig;
     this.communicationConfig = embark.config.communicationConfig;
     // TODO get options from config instead of options
-    this.locale = options.locale;
+    this.locale = embark.locale;
     this.logger = embark.logger;
-    this.client = options.client;
-    this.isDev = options.isDev;
+    this.client = embark.client;
     this.events = embark.events;
-    this.plugins = options.plugins;
+    this.isDev = (this.blockchainConfig.isDev || this.blockchainConfig.default);
```

Comment:
What is `this.blockchainConfig.default`?

Reply:
it was taken from here https://github.com/embarklabs/embark/pull/2208/files#diff-a7c4cef8bfebeb39fcd092aca5570fecL100

Reply:
Ah ok, I think it doesn't make sense and we should probably remove that part

Reply:
Just reiterating that we should remove `this.blockchainConfig.default`

Reply:
I thought we got rid of `isDev` in changes above?

Reply:
after discusisng this with @jrainville 

there is a line in config.ts:
```
if (!configFilePath) {
  this.blockchainConfig.default = true;
}
```

so it seems to be there to put isDev as true when no config is given, stil lit feels weird and it need to rellook at it when refactoring the config